pkg/rancher-desktop/integrations/__tests__/pathManager.spec.ts

```diff
@@ -4,6 +4,7 @@ import path from 'path';
 
 import { START_LINE, END_LINE } from '@pkg/integrations/manageLinesInFile';
 import { RcFilePathManager } from '@pkg/integrations/pathManager';
+jest.mock('../../main/mainEvents');
```

Comment:
I'm researching why using the `@pkg` alias for the jest mocks isn't working. The  relative path appears to be working just fine for now.